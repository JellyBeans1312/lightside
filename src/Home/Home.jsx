import React from 'react';
import './Home.scss';
import PropTypes from 'prop-types'

const Home = ({crawl}) => {
  return(
    <section className='home'>
      <div className="fade"></div>
      <section className="star-wars">
        <div className="crawl">
          <div className="title">
            <h1>{crawl.title}</h1>
          </div>
          <p>{crawl.opening_crawl}</p>
          <p>{crawl.release_date}</p>
        </div>
      </section>
    </section>
    
  )
}

Home.propTypes = {
  crawl: PropTypes.object
}

export default Home;