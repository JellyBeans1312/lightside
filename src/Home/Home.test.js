import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

describe('Home', () => {
  let wrapper
  let mockCrawl;
  beforeEach(() => {
    mockCrawl= {
      title: 'hello',
      opening_crawl: 'hi',
      release_date: 239
    }
    wrapper = shallow(
      <Home 
        crawl={mockCrawl}
      />
    )
  });

  it('should match snapshots', () => {
     expect(wrapper).toMatchSnapshot()
  })
});