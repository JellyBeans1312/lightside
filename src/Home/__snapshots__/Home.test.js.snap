// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match snapshots 1`] = `ShallowWrapper {}`;
